apps/zoho/README.md

```diff
@@ -1,2 +1,40 @@
 # Zoho Hypersync App Sample
-This custom Hypersync app sample allows you to connect to your Zoho tenant and extract user information.  It can be easily extended to retrieve additional information from Zoho.
+This custom Hypersync app sample allows you to connect to your Zoho tenant and extract various pieces of Zoho information.  It can be easily extended to retrieve additional information from Zoho.
+
+## OAuth Configuration
+The Zoho custom app uses OAuth 2.0 for authentication.  The OAuth samples in this repository require some upfront configuration to work properly in a Hyperproof organization.  The steps below will help you configure the Zoho app in your organization.
+
+1. Deploy the Zoho app to your organization using the HP CLI:
+```
+hp customapps import -d .
+```
+ 
+2. After the deployment is complete, run the following command to get the ID of your deployed Zoho app:
+ ```
+ hp customapps list
+ ```
+
+3. Now that you have the unique ID of your app you can use the `getredirect` command to get the OAuth 2.0 redirect URL that is unique to your application:
```

Comment:
If we let people reference their custom apps using the package name steps 2 and 3 could be combined.  But that's an incremental improvement IMO.